www/layouts/partials/home_featured_course_cards.html

```diff
@@ -0,0 +1,83 @@
+{{- $staticApiBaseUrl := getenv "STATIC_API_BASE_URL" | default "https://ocwnext.odl.mit.edu"  -}}
+{{- $featuredCourses := slice -}}
+
+<!-- getting featured courses from course lists whose title is featured-courses -->
+{{- range where $.Site.Pages "Params.title" "featured-courses" -}}
+  {{ if eq .Params.content_type "course-lists" }}
+    {{ $featuredCourses = .Params.courses }}
```

Comment:
If your suggestion is to keep this code block: 
```
  {{ if eq .Params.content_type "course-lists" }}
    {{ partial "featured_course_cards" . }}
  {{end}}
```
in another partial to make `featured_course_cards.html` more reusable, I think we can also move this code block outside into `home.html` as something similar has also been done in the case of `home_new_course_cards.html`.

What do you think?

btw, I think there should be only one partial whose job will be to take courses and show them in a carousel but since `home_new_course_cards.html` looked a bit tricky and time taking to modify so I created a new and separate partial for featured courses, however, both of these partials have some same HTML code. I think we can improve/refactor this later on.

Reply:
That sounds good to me